### PR TITLE
VACMS-9160: Add covid status preview to facility form.

### LINKED ISSUE #9160: Display a preview of the Veteran experience of Facility COVID status within the Faclity form
## User Story or Problem Statement

As an Facility editor, i want to see how the Veteran experiences Low/Medium/High for COVID status. 

<details><summary>Design</summary>
​
Screen Shot 2022-05-24 at 11 04 24 AM

[Figma](https://www.figma.com/file/W4oO1UO0ilSgHu1kXf1Sj0/VAMC-facilities---COVID-status?node-id=0%3A1)

</details>

## Acceptance Criteria
- [ ] When an editor selects a COVID status, they get a preview of the Veteran experience, per the design. 
- [ ] No linked timestamp, for now.

## Implementation steps
<optional>

## Design principles

Veteran-centered
- [ ] `Single source of truth`: Increase reliability and consistency of content on VA.gov by providing a single source of truth.
- [ ] `Accessible, plain language`: Provide guardrails and guidelines to ensure content quality.
- [ ] `Purposely structured content`: Ensure Content API can deliver content whose meaning matches its structure.
- [ ] `Content lifecycle governance`: Produce tools, processes and policies to maintain content quality throughout its lifecycle.

Editor-centered
- [ ] `Purpose-driven`: Create an opportunity to involve the editor community in VA’s mission and content strategy goals.
- [ ] `Efficient`: Remove distractions and create clear, straightforward paths to get the job done.
- [ ] `Approachable`: Offer friendly guidance over authoritative instruction.
- [ ] `Consistent`: Reduce user’s mental load by allowing them to fall back on pattern recognition to complete tasks.
- [ ] `Empowering`: Provide clear information to help editors make decisions about their work.

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Platform CMS Team`
- [ ] `Sitewide program`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
closes #9160

## Testing done
Visual / behat

## Screenshots
image

## QA steps
**Test 1**
 - [x] As an administrator, go to `node/1524/edit` and visually verify Guidelines intro and description below covid status radio buttons
 - [x] Verify box is filled with text on load
 - [x] Verify tooltip displays tooltip message on hover
 - [x] Verify changing radio buttons changes text in tooltip box
 - [x] Verify saving node captures changes made to covid status radios
 
 **Test 2**
 - [x] As an administrator, go to `node/add/health_care_local_facility` and visually verify Guidelines intro and description below covid status radio buttons
 - [x] Verify box is filled with text on load
 - [x] Verify tooltip displays tooltip message on hover
 - [x] Verify changing radio buttons changes text in tooltip box
 - [x] Verify saving node captures changes made to covid status radios

### Definition of Done

- [x] Documentation has been updated, if applicable.
- [x] Tests have been added if necessary.
- [x] Automated tests have passed.
- [x] Code Quality Tests have passed.
- [x] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS Team`
- [ ] `Sitewide program`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public websites`
- [x] `⭐️ Facilities`
- [ ] `⭐️ User support`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: config/sync/node_link_report.settings.yml [hunk 1]
```diff
@@ -6,17 +6,21 @@ display_on_node_preview: node_preview
 additional_domains_as_internal: www.va.gov
 domains_to_skip: |-
   9health365.org
+  acemapp.org
```
Comment (JayDarnell) [inline config/sync/node_link_report.settings.yml:9]: I suspect this file was added to the PR accidentally.

**Review by JayDarnell — CHANGES_REQUESTED** [review]
Assuming the node_link yml was not intended to be a part of this PR it should be removed.

File: config/sync/node_link_report.settings.yml (hunk 1 above)
Comment (ethanteague) [reply config/sync/node_link_report.settings.yml:9]: I've gotten into the habit of letting these node_link_report.settings.yml changes go in as they appear to keep things in sync / not have to worry about it in a dedicated pr. If you aren't feeling it, lemme know and I'll stop doing this / pull this out, @JayDarnell

**Review by JayDarnell — APPROVED** [review]

**Comment by swirtSJW** [comment]
A bit disappointed that this was so covid specific.  The machine names and logic should not have been covid at all.

**Pull request merged**